qelem/lib/screens/profile/ChangePassword.dart

```diff
@@ -0,0 +1,139 @@
+              validator: (value) {
+                if (value!.isEmpty) {
+                  return 'Please enter your new password';
+                }
+              },
+              decoration: InputDecoration(
+                  labelText: 'New Password',
+                  suffixIcon: IconButton(
+                      icon: Icon(
+                          newHidden ? Icons.visibility : Icons.visibility_off),
+                      onPressed: () {
+                        setState(() {
+                          newHidden = !newHidden;
+                        });
+                      }),
+                  labelStyle: GoogleFonts.comfortaa(textStyle: textStyle),
+                  border: OutlineInputBorder(
+                      borderRadius: BorderRadius.circular(5.0))),
+            ),
+          ),
+          Padding(
+            padding:
+                const EdgeInsets.symmetric(vertical: 10.0, horizontal: 15.0),
+            child: TextFormField(
+              style: textStyle,
+              obscureText: confirmHidden,
+              controller: confirmPasswordController,
+              validator: (value) {
+                if (value!.isEmpty) {
+                  return 'Please confirm your current password';
+                } else if (value != newPasswordController.text) {
+                  return 'Passwords don\'t match';
+                }
+              },
+              decoration: InputDecoration(
+                  labelText: 'New Password (again)',
+                  suffixIcon: IconButton(
+                      icon: Icon(confirmHidden
+                          ? Icons.visibility
+                          : Icons.visibility_off),
+                      onPressed: () {
+                        setState(() {
+                          confirmHidden = !confirmHidden;
+                        });
+                      }),
+                  labelStyle: GoogleFonts.comfortaa(textStyle: textStyle),
+                  border: OutlineInputBorder(
+                      borderRadius: BorderRadius.circular(5.0))),
+            ),
+          ),
+          Container(
+              margin: const EdgeInsets.only(right: 15.0, top: 15.0),
+              child: Align(
+                alignment: Alignment.centerRight,
+                child: SizedBox(
+                  width: 115.0,
+                  height: 45.0,
+                  child: MaterialButton(
+                    shape: RoundedRectangleBorder(
+                        borderRadius: BorderRadius.circular(5)),
+                    child: Text(
+                      'UPDATE',
+                      style: GoogleFonts.comfortaa(
+                          color: Colors.white,
+                          textStyle: textStyle,
+                          fontSize: 16,
+                          letterSpacing: 2),
+                    ),
+                    color: const Color.fromRGBO(98, 0, 238, 1),
+                    onPressed: () async {},
+                  ),
```

Comment:
You can use ElevatedButton instead. It is much simpler.
```suggestion
                  child: ElevatedButton(
                    child: Text(
                      'UPDATE',
                      style: Theme.of(context).primaryTextTheme.button,
                    ),
                    onPressed: () async {},
                  ),
```